actor/builtin/miner/miner.go

```diff
@@ -121,14 +117,20 @@ type Ask struct {
 
 // State is the miner actors storage.
 type State struct {
+	// Owner is the address of the account that owns this miner. Income and returned
+	// collateral are paid to this address. This address is also allowed to change the
+	// worker address for the miner.
 	Owner address.Address
 
+	// Worker is the address of the worker account for this miner.
+	// This will be the key that is used to sign blocks created by this miner, and
```

Comment:
> This will be the key that is used to sign blocks created by this miner,

Should this read: 

> This actor will own the key that is used to sign blocks created by this miner,

?

Reply:
This is  not the best wording. But an address is a hashed public key and we use them as such.

Reply:
Ah, disregard. I misunderstood what an `Address` was.